app/views/main.py

```diff
@@ -22,21 +22,21 @@
 logger = logging.getLogger(__name__)
 
 
-root_blueprint = Blueprint('root', __name__)
```

Comment:
I thought that root was referring to the root of the url path rather than access